### PR TITLE
LEP-105 Run CCQ E2E tests inside CFE pipeline with latest codebase

### PR DESCRIPTION
**Pull request by starswan**
https://dsdmoj.atlassian.net/browse/LEP-105

Draft for testing - run CCQ tests against the current branch

### CODE REVIEW

**Comment by davidread** [comment]
I think one of you might have added "CCQ_ECR_ENDPOINT" env var into CircleCI, but no longer need it? if so, this is your reminder to delete it

**Comment by exonian** [comment]
I don't think I added this one, so I'll leave it for Stephen

File: .circleci/config.yml [hunk 1]
```diff
@@ -124,6 +150,30 @@ jobs:
...
+          command: |
+            bundle install --without=development
+      - run:
+          name: Setup CCQ database and test env
+          working_directory: ~/project/ccq
+          command: |
+            bundle exec rails db:schema:load
+            bundle exec rake test:prepare
+      - run:
+          name: CFE/CCQ integration specs
```
Comment (davidread) [inline .circleci/config.yml:172]: one more ref to "integration" for you to change ;)

**Comment by davidread** [comment]
Any chance of a bit more output from the running of the tests? e.g. test names

Screenshot 2023-05-12 at 17 25 50

**Comment by starswan** [comment]
> Any chance of a bit more output from the running of the tests? e.g. test names
> 
> Screenshot 2023-05-12 at 17 25 50

Yep - now using 'documentation' output format in the end2end specs so that test names are visible

**Review by LAA-NeilH — APPROVED** [review]

**Pull request merged**